app/controllers/detainees/images_controller.rb

```diff
@@ -1,6 +1,5 @@
 module Detainees
   class ImagesController < ApplicationController
-    respond_to :jpg
```

Comment:
Should be in the commit message for that change. The respond_to method was only available because of Devise adding in the responders gem.